### PR TITLE
CATROID 435. Changing "Looks" brick category to "Look".

### PR DESCRIPTION
**Pull request by InvincibleKnight**
Made changes in the the:
 catroid/src/main/res/values/strings.xml file.
 catroid/src/main/res/values-en-rCA/strings.xml
 catroid/src/main/res/values-en-rAU/strings.xml
 catroid/src/main/res/values-en-rGB/strings.xml
files.

### CODE REVIEW

**Comment by InvincibleKnight** [comment]
> Thanks! Why did you make changes to the _Project.xml_?

I  did not intend to make changes in Projects.xml

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -8,9 +8,6 @@
     <option name="LINE_SEPARATOR" value="&#10;" />
     <option name="WRAP_WHEN_TYPING_REACHES_RIGHT_MARGIN" value="true" />
     <option name="FORMATTER_TAGS_ENABLED" value="true" />
-    <AndroidXmlCodeStyleSettings>
-      <option name="USE_CUSTOM_SETTINGS" value="true" />
-    </AndroidXmlCodeStyleSettings>
```
Comment (InvincibleKnight) [inline .idea/codeStyles/Project.xml]: Is it okay if add these lines again?

**Comment by InvincibleKnight** [comment]
I've been stuck on this for quite sometime. I do not know how and why there are unintended changes in the Project.xml file. @m-herold, what should my course of action be? Please help.
Thank you.
Darshit.

File: .idea/codeStyles/Project.xml (hunk 1 above)
Comment (m-herold) [reply .idea/codeStyles/Project.xml]: Yes, you can restore the original file. As a rule of thumb I am suggesting to never submit any files you did not intentionally change. Always have a look at the "Files changed" tab before submitting a PR.

Comment (InvincibleKnight) [reply .idea/codeStyles/Project.xml]: This branch of my repository was completely botched. Please check this [PR](https://github.com/Catrobat/Catroid/pull/3555). Thank You.
Cheers,
Darshit.

**Pull request closed without merging**